### PR TITLE
unittest

### PR DESCRIPTION
**Pull request by minhquang31011998**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by minhquang31011998** [comment]
Screenshot from 2020-11-20 15-59-19
Screenshot from 2020-11-20 15-59-01

**Comment by minhquang31011998** [comment]
ready

File: tests/Unit/Http/Controllers/UserControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        $this->userMock
+            ->shouldReceive('changePassword')
+            ->once()
+            ->andReturn(true);
+
+        $data = [
+            'password' => $password,
+        ];
+        $result = $this->userController->sendEmailForgotPassword($requestData);
+        $this->assertEquals('auth.email_success', $result->getName());
```
Comment (huyenttm-0901) [inline tests/Unit/Http/Controllers/UserControllerTest.php:68]: em gọi cái này nó có send email cho em k?

File: tests/Unit/Http/Controllers/UserControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        ];
+        $requestData = new Request($request);
+
+        $this->userMock
+            ->shouldReceive('findUserByEmail')
+            ->with($requestData)
+            ->once()
+            ->andReturn(null);
+
+        $result = $this->userController->sendEmailForgotPassword($requestData);
```
Comment (huyenttm-0901) [inline tests/Unit/Http/Controllers/UserControllerTest.php:84]: tương tự

File: tests/Unit/Http/Controllers/UserControllerTest.php (hunk 1 above)
Comment (minhquang31011998) [reply tests/Unit/Http/Controllers/UserControllerTest.php:68]: ko ạ

**Comment by minhquang31011998** [comment]
ready

**Pull request merged**